dao/src/main/java/com/epam/esm/dao/config/DBConfig.java

```diff
@@ -5,25 +5,67 @@
 import org.springframework.context.MessageSource;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
-import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Profile;
 import org.springframework.context.support.ReloadableResourceBundleMessageSource;
+import org.springframework.core.env.Environment;
 import org.springframework.jdbc.core.JdbcTemplate;
-import org.springframework.jdbc.datasource.DataSourceTransactionManager;
+import org.springframework.orm.jpa.JpaTransactionManager;
+import org.springframework.orm.jpa.JpaVendorAdapter;
+import org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean;
+import org.springframework.orm.jpa.vendor.HibernateJpaVendorAdapter;
+import org.springframework.stereotype.Repository;
 import org.springframework.transaction.PlatformTransactionManager;
 
+import javax.annotation.Resource;
 import javax.sql.DataSource;
+import java.util.Objects;
+import java.util.Properties;
 
-@Configuration
+@Repository
 @ComponentScan(basePackages = "com.epam.esm")
 public class DBConfig {

```

Comment:
Actually, with SpringBoot this configuration class is redundant. Maybe, it should be used only for declaring MessageSource bean. Specifying required properties in application.properties/application.yml is enough to set up desired settings and the required beans should be provided by Boot out of the box.

Reply:
thanks, changed to configuration via application.properties